### ISSUE TITLE
Competitive analysis

### ISSUE THREAD

**Issue by lizzieamanning** [CDCgov/prime-reportstream#9093]
## User story
As a user, I want to experience the ReportStream brand in a consistent and recognizable way.

## Background & context
Create clear, consistent and excellent content across channels by defining guidelines for messaging, tone and editorial standards that will increase our credibility with users.

## Open questions
What's the best format for this analysis?
- Previously started by using .pdf format since the format was left open, but after feedback updated it to a spreadsheet. 

## Working links
[Planning doc from #9092](https://docs.google.com/document/d/1lo83M6Ahw8SRx56xsZVp1wvE3QHvee_m2dBiBYCrajo/edit#)
[Competitive Analysis](https://docs.google.com/spreadsheets/d/1reAgSTwTgRcuQFyAdwMPb0-MxCWc0YVE6sUCquZEA-U/edit#gid=0)

## Acceptance criteria

- [x] Conduct the analysis based on products and criteria defined in #9092
- [x] Create a spreadsheet with categories to track across four companies 
- [x] Review with the design team
- [x] Make updates based on feedback from design team

**Comment by lizzieamanning** [comment]
@yudonoodles thanks- I took a look. Let's chat about this in our 1:1 tomorrow.

**Comment by yudonoodles** [comment]
@lizzieamanning Can you confirm if the format of how I captured the notes work for you? I am using the same for others.

**Comment by Andrey-Glazkv** [comment]
Meeting with Lizzie on 5/9
Tshering to update the ticket after the call

**Comment by yudonoodles** [comment]
cc: @lizzieamanning

**Comment by yudonoodles** [comment]
@Andrey-Glazkv  Met with @lizzieamanning yesterday, and we aligned on using a table format for presenting the comparative analysis. We also added a few more details to track besides what I was tracking. 

@chris-kuryak You had a qn on deliverables (see above). Since I aligned with Lizzie, we no longer need to have the other related ticket #9436 that separated out the work. Andrey helped remove that ticket, we closed it since there isn't a way to delete. 

@Andrey-Glazkv and I also spoke about how to scope future tickets, i.e using an epic and adding separate smaller chunks of stories/tasks that are more defined such as "Conduct 3 of x" or "Explore 2 of x." Just noting it here.

**Comment by chris-kuryak** [comment]
@yudonoodles thanks for the update!

**Comment by yudonoodles** [comment]
Noting additional alignment on scope change here after call with Lizzie yesterday. Initially the scope of this ticket was about reviewing the websites and gathering what you can on them by gleaning through them (as laid out in the competitive analysis by Audrey Ross [planning doc](https://docs.google.com/document/d/1lo83M6Ahw8SRx56xsZVp1wvE3QHvee_m2dBiBYCrajo/edit#heading=h.v08u4dnq73d0)

The scope for what to use for research expanded to now reading through this resource [DIBBs Wiki](https://docs.google.com/document/d/1h-GhNk3yR9xGqq7Oh1U9ZWzhdw0broNr6lKe8lQmm2g/edit#) in order review each product in more detail and capture the insights in our google sheet. We determined that the websites are not enough to understand what the product is, how it works and the finer technical details on what is different or similar to RS. 

cc: @lizzieamanning

**Comment by yudonoodles** [comment]
Today, showing the spreadsheet with design team to get feedback on it.

**Comment by yudonoodles** [comment]
Initially the request for this was to be not so comprehensive, but reviewing with Lizzie on 5/09, the list of things we are tracking grew from previously defined to also include a few more things:
- Type of System (descriptive)
- Type of Messages supported (descriptive)
- Relationship to RS (descriptive)
- Processing steps (descriptive)
- Open Source or Proprietary (yes/no)

After reviewing the [spreadsheet](https://docs.google.com/spreadsheets/d/1reAgSTwTgRcuQFyAdwMPb0-MxCWc0YVE6sUCquZEA-U/edit#gid=0) with the design team (design crit) on 05/11 the following updates and new additions are to be made as tracked through this [crit document](https://docs.google.com/document/d/1Gh5Wx1AqLewg9FBMJP6VPbtv_8ggPqsUWoXy6I1PSYQ/edit#heading=h.drow5wt5jtql): 
- Adding more columns with yes/no for each type of format supported eg. HL7 vs FHIR vs XML
- Two new columns: Limitations and Advantages
- Observations 
- Potentially no. of senders (if that information is knowable via DIBBs wiki and website)

Overall, team noted that this format is great for scanning and comparing the four companies against. cc: @lizzieamanning

**Comment by Andrey-Glazkv** [comment]
Tshering, thank you for updating this @yudonoodles

**Comment by yudonoodles** [comment]
@lizzieamanning I added a few more details by going through brochures I found on MirthConnect and AIMs (their resources are in pdf format) as well as reviewed and added in more information for SimpleReport, especially for the Observation column, which is new. 

To reiterate, scope for discovery was create a spreadsheet comparing four companies using two sources: 1. landing pages of the websites and 2. the DIBBs wiki. 

As I mentioned during our morning 1-1 call, I filled out information is knowable based on those sources. I think as a future to-do, the design team could make some updates of this artifact as more information for some of the categories defined are known such as what the "Advantages" and "Limitations" are or costs associated. 

Related to the outlined final step from my last update, I cleaned up the table and added formatting as requested by the design team.

This ticket is ready to be closed.
